core/src/main/kotlin/knote/Main.kt

```diff
@@ -5,13 +5,8 @@ import mu.KLogging
 object Main : KLogging() {
     @JvmStatic
     fun main(vararg args: String) {
-        KNote.NOTEBOOK_MANAGER.notebookFilter = args.toList()
-        KNote.NOTEBOOK_MANAGER.evalNotebooks()
-
-        KNote.NOTEBOOK_MANAGER.notebooks.forEach { (id, notebook) ->
-            notebook.pageManager!!.executeAll().forEach { pageId, result ->
-                logger.info("[$pageId]: KClass: ${result::class} value: '$result'")
-            }
+        KNote.NOTEBOOK_MANAGER.notebook!!.pageManager!!.executeAll().forEach { pageId, result ->
+            logger.info("[$pageId]: KClass: ${result::class} value: '$result'")
```

Comment:
afaik this WILL crash only when the noteook and scripts cannot compile when it starts,
for the commandline app that is fine i think, but i can also use the proper methods for getting a pageManager i think